packages/web-ui/package.json

```diff
@@ -45,13 +45,15 @@
     "react": "18.3.0",
     "react-dom": "18.3.0",
     "react-resizable-panels": "^2.0.22",
+    "react-textarea-autosize": "^8.5.3",
     "tailwind-merge": "^2.4.0",
     "tailwindcss-animate": "^1.0.7",
     "use-debounce": "^10.0.1",
     "zod": "^3.23.8",
     "zustand": "^4.5.4"
   },
   "devDependencies": {
+    "@faker-js/faker": "^8.4.1",
```

Comment:
I guess this is temporal until real AI. Remember to remove once we don't need it